elastic/kibana
# [ECO][Inventory v2] Hosts entity summary endpoint changes

## Summary

Closes #202300

This PR changes the entity client function to v2 (`searchEntities`) in `getLatestEntity`.  After the change to use `v2.searchEntities` the parameters are also updated to include the time range  (`start` and `end` are required)

## Testing
~- We can create some definitions manually-  in the Kibana DEV tools: ~ - Not needed after we merged the V2 PR 

- In a local environment enable the entities feature flag ( it should be a clean env as the entities should not be enabled before ): 
<img width="1911" alt="image" src="https://github.com/user-attachments/assets/75d6f77d-5039-41ca-80ca-34c3bf99844e" />

- Some hosts and containers are required - oblt cluster/metricbeat or
   - Create hosts using synthtrace: 
       ```
       node scripts/synthtrace infra_hosts_with_apm_hosts --scenarioOpts.numInstances=20
       ```
     - ⚠️ _Update_ to test the logs scenario: 
         ```
          node scripts/synthtrace logs_traces_hosts.ts
         ```
   - Create containers using synthtrace: 
       ```
       node scripts/synthtrace infra_docker_containers.ts
       ```

- In the UI 
   - Open asset details view for hosts and containers and check the summary endpoint response: 
  ⚠️ Updated: 

![image](https://github.com/user-attachments/assets/27683b74-f0b5-43a0-9a8f-98cd2a61e68e)

  - If the entities FF is disabled (default: no `logs` should be part of the `sourceDataStreams`): 

![image](https://github.com/user-attachments/assets/7b8851b4-514c-4fc7-ab84-720b2ccb16ae)
